ush/forecast_postdet.sh

```diff
@@ -809,7 +809,7 @@ MOM6_postdet() {
   esac
 
   # Loop over restart_interval frequency and link restarts from DATA to COM
-  local idate=$(date -d "${current_cycle:0:8} ${current_cycle:8:2} + ${restart_interval} hours" +%Y%m%d%H)
+  local idate=$(date --utc -d "${current_cycle:0:8} ${current_cycle:8:2} + ${restart_interval} hours" +%Y%m%d%H)
   while [[ ${idate} -lt ${forecast_end_cycle} ]]; do
     local idatestr=$(date +%Y-%m-%d-%H -d "${idate:0:8} ${idate:8:2}")
```

Comment:
Should this also have --utc?

Reply:
This is a simple format change and so does not need the `--utc` option.
When datetime is being advanced forward/backward, we should use the `--utc` option.